kronolith/lib/Event.php

```diff
@@ -775,7 +782,9 @@ public function toiCalendar($calendar)
         $vEvent->setAttribute('SUMMARY', $this->getTitle());
 
         // Organizer
-        if (count($this->attendees)) {
+        if ($this->organizer) {
+            $vEvent->setAttribute('ORGANIZER', 'mailto:' . $this->organizer, array());
+        } elseif (count($this->attendees)) {
```

Comment:
Why elseif? The attendees should be added if they exist, not if the organizer doesn't exist.